Sources/web3swift/Transaction/EIP1559Envelope.swift

```diff
@@ -117,8 +106,6 @@ extension EIP1559Envelope {
         case v
         case r
         case s
-        // MARK: workaround for gasPrice coming from nodes for EIP-1159 - this allows Oracle to work for now
```

Comment:
It's 1559 mistyped to 1159

Reply:
indeed, but it's a line being deleted, so no need to fix.